java-desktop/src/main/java/com/couchbase/todo/model/Config.java

```diff
@@ -0,0 +1,109 @@
+package com.couchbase.todo.model;
+
+import java.util.Objects;
+
+import com.couchbase.lite.ConsoleLogger;
+import com.couchbase.lite.Database;
+import com.couchbase.lite.LogDomain;
+import com.couchbase.lite.LogLevel;
+import com.couchbase.todo.TodoApp;
+
+
+public final class Config {
```

Comment:
Ok.... This seems like a reasonable solution but I'm going to ask you to try something else.  First you have to read this:
https://sites.google.com/site/steveyegge2/singleton-considered-stupid
Then I'd like you to re-do this so that the ToDo application holds the current, immutable, config.

We can talk about this if you'd like.

The change won't require much change in the code... this is almost there.  But I'd like to see all of the state variables be final.

Reply:
Looks really good.